OWNERS

```diff
@@ -1,18 +1,22 @@
 reviewers:
-  - brancz
-  - metalmatze
-  - s-urbaniak
-  - squat
-  - LiliC
-  - bwplotka
-  - pgier
-  - simonpasquier
+- brancz
```

Comment:
hmm, is the indentation right?

Reply:
yeah, for lists in yaml it's not required to indent.  I just copied the file contents from the openshift repo (https://github.com/openshift/prometheus-operator/blob/master/OWNERS).  You can blame @simonpasquier for changing the indent :)